### DISCUSSION TITLE
Webviewer, document.getElementByID('id') returns ("") emptystring. Should be obj{} i think.

### DISCUSSION THREAD

**Discussion by Daantje007** [unoplatform/uno#8946] [Q&A]
WebViewer:
            
var functionString2 = string.Format("document.getElementById('content').style.width = '1500px'");
await appwebview.InvokeScriptAsync("eval", new string[] { functionString2 });

Seems to work with getElementByTagName hmm..
What am i missing? 
Thank you,

**Comment by jeromelaban** [comment]
We'll need a bit more information here. Can you tell which platform you are trying to run on? On which version of Uno?

**Comment by Daantje007** [comment]
For now just trynig UWP i understand its different for WASM but do i read here, its not suported for UWP at all??
[https://platform.uno/docs/articles/implemented/windows-ui-xaml-controls-webview.html](url)
Thanks J,

**Reply by jeromelaban** [reply]
For UWP, you're likely hitting something specific to your javascript page, you can debug this [with standard tooling](https://docs.microsoft.com/en-us/previous-versions/visualstudio/visual-studio-2017/debugger/debug-a-webview-control?view=vs-2017).

Wasm [is different](https://platform.uno/docs/articles/interop/wasm-javascript-1.html), as the browser is already a webview in itself.

**Reply by Daantje007** [reply]
But the document.getElementById('..') is troubleing me, the other javascript functions work fine..  i will try again tomorow.. How can i check my current uno version te best way? i dont think you mean the uno nugets? i use default temp based on xamarin

**Reply by Daantje007** [reply]
![image](https://user-images.githubusercontent.com/95089394/171290257-2a2baa83-6a18-4a8f-8623-4fc015710ac2.png)

**Reply by Daantje007** [reply]
![image](https://user-images.githubusercontent.com/95089394/171290467-87b66f52-a0e0-4aeb-b6e2-5039cd09ccac.png)

**Reply by Daantje007** [reply]
First one is xamarin
Seccond is Uno Based on Xamarin

**Reply by Daantje007** [reply]
And i have no other function that comes close to EvaluateJavaScriptAsync() i hoped they would do the samething XD

**Reply by Daantje007** [reply]
Interesting for this topic
[https://stackoverflow.com/questions/38432594/how-to-invoke-javascript-in-webview-windows-10-uwp](url)
